ray_on_golem/server/services/golem/golem.py

```diff
@@ -53,7 +53,7 @@ async def init(self, yagna_appkey: str) -> None:
         await self._golem.start()
 
         self._network = await self._golem.create_network(
-            "192.168.0.1/24"
+            "192.168.0.1/16"
```

Comment:
why not 8, as discussed?

Reply:
`192.168.0.0` doesn't allow `8` and `16` is more then enough